web/packages/teleport/src/Roles/RoleEditor/StandardEditor/MetadataSection.tsx

```diff
@@ -55,14 +57,25 @@ export const MetadataSection = ({
         onChange({ ...value, description: e.target.value })
       }
     />
-    <Text typography="body3" mb={1}>
-      Labels
-    </Text>
-    <LabelsInput
-      disableBtns={isProcessing}
-      labels={value.labels}
-      setLabels={labels => onChange?.({ ...value, labels })}
-      rule={precomputed(validation.fields.labels)}
+    <Box mb={3}>
+      <Text typography="body3" mb={1}>
+        Labels
+      </Text>
+      <LabelsInput
+        disableBtns={isProcessing}
+        labels={value.labels}
+        setLabels={labels => onChange?.({ ...value, labels })}
```

Comment:
`onChange` is optional? Why?

Reply:
I originally did it to make it easier to instantiate these components in tests, but now I see it wasn't needed at all. Making it more strict.